plugins/org.eclipse.elk.alg.packing.rectangles/src/org/eclipse/elk/alg/packing/rectangles/rectPacking.melk

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Kiel University and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Kiel University - initial API and implementation
+ *******************************************************************************/ 
+package org.eclipse.elk.alg.packing.rectangles
+ 
+/* Metadata file for rectangle packing algorithms. Enables support for options and specifies auto-generated files.
+ * 
+ * @author dalu
+ */
+
+import org.eclipse.elk.alg.packing.rectangles.RectPackingLayoutProvider
+import org.eclipse.elk.alg.packing.rectangles.util.PackingStrategy
+import org.eclipse.elk.core.math.ElkPadding
+
+//BUNDLE
+bundle {
+    metadataClass options.RectPackingMetaDataProvider
+}
+
+//ALGORITHM
+algorithm rectPacking(RectPackingLayoutProvider) {
+    label "Rectangle Layout."
+    description "Algorithm for packing of unconnected boxes, i.e. graphs without edges."
+    metadataClass options.RectPackingOptions
+    supports org.eclipse.elk.aspectRatio = 1.3
+    supports org.eclipse.elk.padding = new ElkPadding(15)
+    supports org.eclipse.elk.expandNodes
+    supports strategy
+    supports lpShift
+    supports onlyFirstIteration
+    supports checkForSpecialCase
+}
+
+//OPTIONS
+    advanced option strategy: PackingStrategy {
+        label "Packing Strategy"
+        description 
+            "Strategy for approximation of the bounding box. Determines whether layout is sorted by scale measure, aspect ratio or area."
+        targets nodes
+        default = PackingStrategy.MAX_SCALE_DRIVEN
+    }
+    
+    advanced option lpShift : boolean{
```

Comment:
Add a space before the brace, check for consistency with other melks regarding the colon. 

Also, can you think of a more descriptive name from a user's perspective? Personally, I think of linear programming when reading `lp`.